custom/ewsghana/reminders/first_soh_reminder.py

```diff
@@ -9,7 +10,8 @@ def recipients_filter(self, user):
         roles = user.user_data.get('role', [])
         if not roles:
             return False
-        return any([role != IN_CHARGE_ROLE for role in user.user_data.get('role', [])])
+        needs_reminders = string_to_boolean(user.user_data.get('needs_reminders', "False"))
```

Comment:
I spoke with Mike and we should make it so that in case someone leaves a value out or puts a bad value (with bulk user import for example) we should assume that they receive the reminders. I'm going to merge this to make sure it gets in for Thursday, but can you do another PR to make this line be something like:

`needs_reminders = False if user.user_data.get('needs_reminders') == 'False' else True`